spec/models/book_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+describe Book do
+  let(:book) { build(:book) }
+
+  context 'When creates a book' do
+    it 'validates presence of :gender' do
+      is_expected.to validate_presence_of(:gender)
+    end
+
+    it 'validates presence of :gender' do
+      is_expected.to validate_presence_of(:gender)
+    end
+
+    it 'validates presence of :author' do
+      is_expected.to validate_presence_of(:author)
+    end
+
+    it 'validates presence of :image' do
+      is_expected.to validate_presence_of(:image)
```

Comment:
for this built in validations, we feel there is no need for the it to have a description, since it's already easy to read and if it fails the error is understandable.
For example:
`it { is_expected.to validate_presence_of(:image) }`